ios/FluentUI/Vnext/List/List.swift

```diff
@@ -68,6 +68,21 @@ public struct MSFListView: View {
                 }
             }
     }
+
+    private func dividers(_ sections: [MSFListSectionState]) -> [MSFListSectionState] {
+        var cells: [MSFListCellState]
+        var hasDividers: Bool
+        for sectionIndex in 0...sections.count - 1 {
```

Comment:
It's cleaner to do "for section in sections {" instead of using indexes in this case.

Or you can use the ForEach func for convenience:

  
```
sections.forEach { section in
    section.cells.forEach { cell in
        cell.hasDivider = section.hasDividers
    }
}
```